src/PrestaShopBundle/Resources/config/services/core/hook.yml

```diff
@@ -6,3 +6,83 @@ services:
         class: 'PrestaShop\PrestaShop\Core\Hook\HookDispatcher'
         arguments:
             - '@prestashop.hook.dispatcher'
+
+    prestashop.core.hook.provider.grid_definition_hook_by_service_ids_provider:
```

Comment:
Maybe we could define them only in `config_dev.xml` 🤔 ? Again I dont whether this really harms container performance.

Reply:
for me it seems pretty good idea to separate code according to environment - shall we start implementing your suggestion within this PR ? :thinking:

Reply:
Let's keep it as an idea for now.
@mickaelandrieu If we wanted to load some services only for DEV environment
- would you say it's a good idea or is it going to introduce too much complexity ?
- would you do it by creating a `src/PrestaShopBundle/Resources/config/services/core/hook_dev.yml` file and load it only from `config_dev.yml` ?